README.md

```diff
@@ -1,5 +1,7 @@
 # ember-toastr
 
+This plugin must not be used anymore, [see EP-688](https://bottomline.atlassian.net/browse/EP-688). The repository is kept to enable building older APKs that require this plugin.
```

Comment:
I would not mind having this bold-faced or in some other format to draw attention.